packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -376,8 +376,8 @@ class FlameChartItem {
     });
   }
 
-  final defaultTextColor = isDarkTheme ? Colors.white : Colors.black;
-  final selectedTextColor = isDarkTheme ? Colors.black : Colors.white;
+  final defaultTextColor = defaultBackground.reverse();
```

Comment:
instead of calling reverse, define additional colors. Someone tweaking defaultBackground will not expect that the color they specified for the dark theme is actually used in a light theme, etc.

Reply:
done.